code/entities/weapons/fnb/Popcorn.cs

```diff
@@ -1,12 +1,18 @@
-﻿using Sandbox;
+﻿using Conna.Inventory;
```

Comment:
There were no substantial changes in this file. Please remove it from the changes.